sentinel.go

```diff
@@ -553,9 +591,9 @@ func (c *sentinelFailover) getSlaveAddrs(ctx context.Context, sentinel *Sentinel
 	return parseSlaveAddrs(addrs)
 }
 
-func parseSlaveAddrs(addrs []interface{}) []string {
+func parseSlaveAddrs(addrs []interface{}, allowedFlags ...string) []string {
```

Comment:
If I read this code correctly `allowedFlags` should be replaced with `keepDisconnected bool`.